keel-test/src/main/kotlin/com/netflix/spinnaker/keel/test/Resources.kt

```diff
@@ -0,0 +1,107 @@
+package com.netflix.spinnaker.keel.test
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.keel.api.ApiVersion
+import com.netflix.spinnaker.keel.api.Monikered
+import com.netflix.spinnaker.keel.api.ResourceSpec
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.ResourceKind
+import com.netflix.spinnaker.keel.api.SPINNAKER_API_V1
+import com.netflix.spinnaker.keel.api.randomUID
+import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import com.netflix.spinnaker.keel.plugin.ResourceNormalizer
+import com.netflix.spinnaker.keel.serialization.configuredObjectMapper
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+val TEST_API = SPINNAKER_API_V1.subApi("test")
+
+fun resource(
+  apiVersion: ApiVersion = TEST_API,
+  kind: String = "whatever",
+  name: String = randomString(),
+  application: String = "fnord"
+): Resource<DummyResourceSpec> =
```

Comment:
First variant of test factory, lets you easily build a `Resource<DummyResourceSpec>` which is probably ideal for most tests that are not dealing with a specific resource type. You can call this with no parameters to get a nice random resource.